### DISCUSSION TITLE
JSON feed meeting list (on Area website) is different than what's displayed on the Meeting Guide app

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#1284] [Q&A] [answered]
**Dear Code4Recovery friends,**

The meetings list for District 48 (Williamsport, PA) is slightly different when viewed on the Meeting Guide app versus the Area 59 website.

https://area59aa.org/find-a-meeting/?region=williamsport&search=sober&weekday=sunday

When viewed on the Meeting Guide app, there are (2) listings for the same meeting (with different addresses). (see attached)

[A59-District-48-Meetings-Inconsistency-DEC-15-2023.pdf](https://github.com/code4recovery/12-step-meeting-list/files/13720385/A59-District-48-Meetings-Inconsistency-DEC-15-2023.pdf)

Area 59 website uses C4R Google Sheets Importer for their D48 Google Sheet to create a JSON file:

A59-D48-MeetingGuide-MASTER:

https://docs.google.com/spreadsheets/d/1fJINOisOEM7Fpv5gTOAPnmkPi6NuBWkZ95W72FemmQI/edit?usp=sharing

https://sheets.code4recovery.org

Feed created:

https://sheets.code4recovery.org/storage/1jhAO6X9egDa18Pp04n3NcLp9_CM9gcEu0IVUSCZG0OQ.json

The new D48 webservant did not receive much guidance from its predecessor and has asked if Area 59 could maintain its Google Sheet. I am glad to help, as I'm already doing that with another District in Area 59 that uses the Google Sheets process.

QUESTION: I am unclear where the apparently "old" list is coming from that displays on the Meeting Guide app.

We really appreciate any assistance you're able to provide.

In fellowship,

**Scott Fritzinger**, Area 59 webservant

webservant@area59aa.org

**Jim L.**, District 48 webservant

jlabenberg@gmail.com

**Comment by joshreisner** [comment, marked as answer]
looks like the problem meeting is imported directly from the district, perhaps confirm with gso what url they are importing from for the district and make sure the information there is correct. appsupport@aa.org

or if you are importing their info to to the area, ask them to disconnect the district so the listings will come from the area?

![IMG_0091](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/5e4b0c49-723d-4de6-9c8c-e5ecbcc86935)

**Reply by fritzisa** [reply]
Thanks so much, Josh!
